core/modules/qdisp/Executive.h

```diff
@@ -124,12 +126,20 @@ class Executive : public std::enable_shared_from_this<Executive> {
     bool xrdSsiProvision(std::shared_ptr<QueryResource> &jobQueryResource,
                          std::shared_ptr<QueryResource> const& sourceQr);
 
+    std::mutex sumMtx; // TEMPORARY-timing
+    int cancelLockQSEASum{0}; // TEMPORARY-timing
+    int jobQueryQSEASum{0}; // TEMPORARY-timing
+    int addJobQSEASum{0}; // TEMPORARY-timing
+    int trackQSEASum{0}; // TEMPORARY-timing
+    int endQSEASum{0}; // TEMPORARY-timing
```

Comment:
Well, if you are going to merge this it's not temporary anymore. Could we instead design more permanent and less exposed instrumentation for performance measurements, I think it would be useful in the long term.